packages/components/src/navigator/navigator-provider/component.tsx

```diff
@@ -75,6 +83,90 @@ function UnconnectedNavigatorProvider(
 	useEffect( () => {
 		currentLocationHistory.current = locationHistory;
 	}, [ locationHistory ] );
+	useEffect( () => {
```

Comment:
There seems to be a bug that causes the component to add the initial location twice to the location history when loading. This also causes the `isInitial` flag added in `navigatorContextValue` to be `false` even when it should be `true`, which causes the initial `NavigatorScreen` to animate on first render (when it shouldn't)